news/2024/2024-12-05-osu-world-cup-2024-finals-recap.md

```diff
@@ -0,0 +1,117 @@
+
+Some thought a historic reverse sweep was in the making, but Germany shattered those dreams, opting into [Eiyuu no Tsurugi](https://osu.ppy.sh/beatmapsets/1864097#osu/3833517) and ending the match with a tight margin of 88,814 points, despite ::{ flag=GB }:: [Bubbleman](https://osu.ppy.sh/users/5182050)'s best efforts with a lobby-best 716k score. It was on to Australia and a shot at the podium for Germany, while the United Kingdom were eliminated with a top 6 placement for the first time since 2021.
+
+The final match of the weekend, with a podium position on the line, was a clash between previous day winners Australia and Germany. Each team had looked dominant in their wins, but the question to be asked was what Germany could do to slow down the MVP-calibre performance of ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902). As Australia sprinted to a 4–0 lead in the match, led by top scores from ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902) on 3 of the 4 maps, the answer seemed to be "nothing", as Germany dropped the first three picks by an average margin of 854k.
+
+Germany finally took their first point on [Hold Your Colour](https://osu.ppy.sh/beatmapsets/702272#osu/1485947), and while they lost two more maps to bring the score to 6–1, ::{ flag=DE }:: [Tim Kackner](https://osu.ppy.sh/users/14385814) put up a Hard Rock FC on [Eiyuu no Tsurugi](https://osu.ppy.sh/beatmapsets/1864097#osu/3833517) to win it, followed by a ::{ flag=DE }:: [criller](https://osu.ppy.sh/users/8116659) and ::{ flag=DE }:: [WhiteCat](https://osu.ppy.sh/users/4504101) masterclass on [Mondaymania](https://osu.ppy.sh/beatmapsets/2284699#osu/4872865) to put up their third point of the match.
+
+Australia, though, would not be denied. It was a double FC from ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902) and ::{ flag=AU }:: [Zyntex](https://osu.ppy.sh/users/11806306) on [roots](https://osu.ppy.sh/beatmapsets/2139515#osu/4502914) to seal the deal, a 7–3 win for the Aussies, sending them onward to the Grand Finals week, a top 3 finish secured for a second year in a row and a date set with familiar nemeses the United States.
```

Comment:
```suggestion
Australia, though, would not be denied. It was a double FC from ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902) and ::{ flag=AU }:: [Zyntex](https://osu.ppy.sh/users/11806306) on [roots](https://osu.ppy.sh/beatmapsets/2139515#osu/4502914) to seal the deal, a 7–3 win for the Aussies, sending them onwards to the Grand Finals week, a top 3 finish secured for a second year in a row and a date set with familiar nemeses in the United States.
```

Reply:
should be `nemeses, the United States`?

Reply:
"nemeses (residing) in the United States"

would be like this with your idea

```suggestion
Australia, though, would not be denied. It was a double FC from ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902) and ::{ flag=AU }:: [Zyntex](https://osu.ppy.sh/users/11806306) on [roots](https://osu.ppy.sh/beatmapsets/2139515#osu/4502914) to seal the deal, a 7–3 win for the Aussies, sending them onwards to the Grand Finals week, a top 3 finish secured for a second year in a row and a date set with a familiar nemesis, the United States.
```

Reply:
https://dictionary.cambridge.org/us/dictionary/english/nemeses checked this and thought it was just plural because they were talking about the members of united states team

Reply:
*if* it refers to the members, that is

"nemeses, the United States" would refer to the plural states. the United States can be a nemesis though